lib/github_api/client/repos/releases/assets.rb

```diff
@@ -35,7 +35,7 @@ def list(*args)
     #
     # @api public
     def get(*args)
-      params = arguments(args, required: [:owner, :repo, :id]).params
+      arguments(args, required: [:owner, :repo, :id]).params
```

Comment:
@piotrmurach If I fix this according to the RuboCop rules now in place, the code will be invalid Ruby for the MRI 1.9.3 that's in the Travis test matrix.